packages/smooth_app/lib/pages/scan/scan_page.dart

```diff
@@ -142,7 +142,7 @@ class _ScanPageTopWidget extends StatelessWidget {
                                 ),
                                 child: Text(
                                   localizations.permission_photo_denied_message(
-                                    localizations.app_name,
+                                    'OpenFoodFacts',
```

Comment:
We definitely need a place to put something like `const String APP_NAME = 'OpenFoodFacts';`
We won't type it every time we need it.

Reply:
Maybe we should rely on `package_info_plus` and the appName?

Reply:
`package_info_plus` makes sense, but I don't think it's worth the efford to wrap this single text around a FutureBuilder or do something similar just to get it to work. Added a const to `design_constants.dart` did not seem appropriate anywhere else